sa/_db/migrations/20160202135920_AddFQDNSetTable.sql

```diff
@@ -0,0 +1,20 @@
+
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+
+CREATE TABLE `fqdnSets` (
+       `id` INT(11) NOT NULL AUTO_INCREMENT,
+       -- SHA256 hash of alphabetically sorted, lowercased, comma joined
+       -- DNS names contained in a certificate
+       `setHash` BINARY(32) NOT NULL,
+       `serial` VARCHAR(255) UNIQUE NOT NULL,
+       `issued` DATETIME NOT NULL,
+       `expires` DATETIME NOT NULL,
+       PRIMARY KEY (`id`),
+       KEY `setHash` (`setHash`)
```

Comment:
Coming back to this: I think we actually want this key to cover (setHash, issued).